build/yamls/antrea-eks.yml

```diff
@@ -286,6 +286,11 @@ data:
     # networkPolicyOnly: Antrea enforces NetworkPolicy only, and utilizes CNI chaining and delegates Pod IPAM and connectivity to primary CNI.
     #
     trafficEncapMode: networkPolicyOnly
+
+    # The port for the antrea-agent to serve on.
+    # Note that if it's set to another value, the `containerPort` of the `api` port of the
+    # `antrea-agent` container must be set to the same value.
+    #bindPort: 10350
```

Comment:
For K8s, it's --bind-address and --secure-port, I think secure-port is because it has an legacy insecure port, so I named it bindPort directly. I'm not sure whether we will have to have other ports, maybe call this one apiPort? do you have suggestion?

Reply:
maybe one of `apiPort`, `apiBindPort`, `apiserverPort`, `apiserverBindPort`?

Reply:
changed to `apiPort`